tests/test_wave_01.py

```diff
@@ -131,13 +134,17 @@ def test_update_task_not_found(client):
     # Assert
     assert response.status_code == 404
 
-    raise Exception("Complete test with assertion about response body")
+    # raise Exception("Complete test with assertion about response body")
     # *****************************************************************
     # **Complete test with assertion about response body***************
     # *****************************************************************
+    assert "details" in response_body
+    assert response_body == {
+        "details": f"Task 1 not found"
```

Comment:
Good job again with the assertion. Using an f-string here is likewise not necessary.